pymc_experimental/tests/test_model_builder.py

```diff
@@ -24,12 +28,13 @@ class test_ModelBuilder(ModelBuilder):
     _model_type = "LinearModel"
     version = "0.1"
 
-    def build_model(self, model_config, data=None):
-
+    def build_model(self, model_instance, model_config, data=None):
+        model_instance.model_config = model_config
```

Comment:
I think we can just keep the `model_config` and `data` in the base class (instance `self`). The init() method should also take care of that.